Compilation error with clang 7

```
/home/pmos/build/src/mir-1.1.2/src/miral/static_display_config.cpp:306:37: error: using integer absolute value function 'abs' when argument is of floating point type [-Werror,-Wabsolute-value]
                                if (abs(conf.refresh.value() - mode->vrefresh_hz) < 1.0)
                                    ^
/home/pmos/build/src/mir-1.1.2/src/miral/static_display_config.cpp:306:37: note: use function 'std::abs' instead
                                if (abs(conf.refresh.value() - mode->vrefresh_hz) < 1.0)
                                    ^~~
                                    std::abs
1 error generated.
```

--- a/src/miral/static_display_config.cpp
+++ b/src/miral/static_display_config.cpp
@@ -303,7 +303,7 @@ void miral::StaticDisplayConfig::apply_to(mg::DisplayConfiguration& conf)
                         {
                             if (conf.refresh.is_set())
                             {
-                                if (abs(conf.refresh.value() - mode->vrefresh_hz) < 1.0)
+                                if (std::abs(conf.refresh.value() - mode->vrefresh_hz) < 1.0)
                                 {
                                     conf_output.current_mode_index = distance(begin(conf_output.modes), mode);
                                     matched_mode = true;

